### ISSUE TITLE
Image for BrainWeb card?

### ISSUE THREAD

**Issue by r03ert0** [nipype/pydra#228]
Hello! It's great that Pydra is in the BrainWeb projects now! \o/
The image in BrainWeb is the first that appears in the Reame.md file, which in Pydra's case is the "build passing" tag. I made an svg and jpg version of your logo with the 3:2 format of BrainWeb. Maybe you'd like to add it to your repo? Then it would just be a matter of linking it in your Readme.md, even with a hidden link like this:

```
<!--(https://raw.githubusercontent.com/nipype/pydra/master/pydra_logo.jpg)-->
```
![pydra_logo](https://user-images.githubusercontent.com/2310732/78387093-075e5180-75df-11ea-9d4d-6bdd630e7005.jpg)

**Comment by satra** [comment]
@r03ert0  - sounds good to me. do you want to send a PR :)

**Comment by r03ert0** [comment]
ok : )

**Comment by satra** [comment]
thank you @r03ert0 - closed via #229
